app/controllers/openid_connect/logout_controller.rb

```diff
@@ -26,6 +26,13 @@ def show
       if result.success? && redirect_uri
         handle_successful_logout_request(result, redirect_uri)
       else
+        if result.extra[:integration_errors].present?
+          analytics.integration_errors_present(
+            **result.
+              to_h[:integration_errors].
+              merge({ event: :oidc_logout_requested }),
```

Comment:
issue: The method `AnalyticsEvents#oidc_logout_requested` tracks the 'OIDC Logout Requested' event. If there is any need to correlate these two events in CloudWatch, this won't work as is.

Reply:
ah great observation -- this event field wasn't intended to correlate in cloudwatch, it was just an indicator to the IEs about where the error is occurring! it felt easiest to use the method name of the event that is firing for discoverability. 

i think if we find in our usage that we'd like to correlate, i can update in later PRs. thanks for planting that seed!